master/internal/kubernetes/pods.go

```diff
@@ -324,6 +329,20 @@ func (p *pods) handleGetAgentsRequest(ctx *actor.Context) {
 func (p *pods) summarize(ctx *actor.Context) map[string]agent.AgentSummary {
 	podHandlers := make([]*actor.Ref, 0, len(p.podNameToPodHandler))
 	for _, podHandler := range p.podNameToPodHandler {
+		metadata := p.podHandlerToMetadata[podHandler]
```

Comment:
blocking: in the commit message by "getPodNodeInfo is a blocking call until all pod actors return a status update." do you actually mean "summarize"?

> "It is possilbe that pod actors are blocked by creating or deleting k8 resources."

Probably be "The pod actors might be blocked for a long time by creating or deleting k8 resources when `summarize` wants a quick response from each pod actor."